templates/content/basic/src/MyProvider.DesignTime/MyProvider.DesignTime.fsproj

```diff
@@ -1,15 +1,10 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
-  <Import Project="..\..\netfx.props" />
   <PropertyGroup>
     <OutputType>Library</OutputType>
-    <TargetFrameworks>netstandard2.0;netcoreapp3.1;net45</TargetFrameworks>
-    <Optimize>true</Optimize>
-    <Tailcalls>true</Tailcalls>
-    <DisableImplicitFSharpCoreReference>true</DisableImplicitFSharpCoreReference>
-    <RestoreProjectStyle>PackageReference</RestoreProjectStyle>
-    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
-    <OutputPath>..\MyProvider.Runtime\bin\$(Configuration)\typeproviders\fsharp41\</OutputPath>
```

Comment:
This output path is crucial (or the output DLL and PDB must somehow be copied to this location).  Without it the type provider package will not have its design time tooling in the right place.  Perhaps this has been moved somewhere else?

Reply:
I've replaced this with a CopyFile in the Runtime .fsproj